### ISSUE TITLE
Plot gravity change no longer working

### ISSUE THREAD

**Issue by mtbell-usgs** [jkennedy-usgs/sgp-gsadjust#38]
[GSadjustLog_20190311-1843.txt](https://github.com/jkennedy-usgs/sgp-gsadjust/files/2954775/GSadjustLog_20190311-1843.txt)

Plot of gravity change is not actually plotting. The text in the conda box is that there's an issue with the axes.

**Comment by jkennedy-usgs** [comment]
It may be an issue with the python environment, as the log file complains about plt.hold which is deprecated in new matplotlib releases. I removed the call to plt.hold, its superfluous.

Also, after loading the workspace referenced in the log file (ABCWUA_All_JK_2019-03-11_MTB.p), I couldn't run the adjustment because some A-10 observations had been imported into the 2016-06-09 survey. Since some of these stations (e.g., SICI) weren't in the network, the adjustment failed. I removed the errant datum observations (from 2018 and 2017), enabled the original observations (without a date), and ran the adjustment. After that I was able to show the gravity-change plot.
